From: Reinhard Tartler <siretart@tauware.de>
Subject: [PATCH] unbreak c++ applications naively including FFmpeg headers

UINT64_C and similar aren't standard in C++ unless this macro is
defined, and ffmpeg headers require it)

--- a/libavutil/avutil.h
+++ b/libavutil/avutil.h
@@ -78,6 +78,11 @@ enum AVMediaType {
     AVMEDIA_TYPE_NB
 };
 
+/* Debian specific hack to unbreak c++ applications naively including FFmpeg headers */
+#ifdef __cplusplus
+#define __STDC_CONSTANT_MACROS
+#endif
+
 #include "common.h"
 #include "error.h"
 #include "mathematics.h"
